src/transactions/feed/JumpstartTransactionDetailsScreen.test.tsx

```diff
@@ -0,0 +1,141 @@
+import { render } from '@testing-library/react-native'
+import React from 'react'
+import { Provider } from 'react-redux'
+import { Screens } from 'src/navigator/Screens'
+import { RootState } from 'src/redux/reducers'
+import { getDynamicConfigParams } from 'src/statsig'
+import JumpstartTransactionDetailsScreen from 'src/transactions/feed/JumpstartTransactionDetailsScreen'
+import {
+  Fee,
+  NetworkId,
+  TokenAmount,
+  TokenTransactionTypeV2,
+  TokenTransfer,
+  TokenTransferMetadata,
+  TransactionStatus,
+} from 'src/transactions/types'
+import {
+  RecursivePartial,
+  createMockStore,
+  getElementText,
+  getMockStackScreenProps,
+} from 'test/utils'
+import {
+  mockAccount,
+  mockCusdAddress,
+  mockCusdTokenId,
+  mockE164Number2,
+  mockJumpstartAdddress,
+} from 'test/values'
+
+jest.mock('src/analytics/ValoraAnalytics')
+jest.mock('src/statsig')
+
+const mockAddress = '0x8C3b8Af721384BB3479915C72CEe32053DeFca4E'
+const mockName = 'Hello World'
+
+describe('JumpstartTransactionDetailsScreen', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+
+    jest.mocked(getDynamicConfigParams).mockReturnValue({
+      jumpstartContracts: {
+        [NetworkId['celo-alfajores']]: { contractAddress: mockJumpstartAdddress },
+      },
+    })
+  })
+
+  function renderScreen({
+    storeOverrides = {},
+    transaction,
+  }: {
+    storeOverrides?: RecursivePartial<RootState>
+    transaction: TokenTransfer
+  }) {
+    const store = createMockStore({
```

Comment:
nit: we don't really need to create a fancy mock store right? the identity and recipients stores are not relevant for this test?